### ISSUE TITLE
Intermittent `WebDriverException: chrome not reachable` error on Windows runners.

### ISSUE THREAD

**Issue by samcunliffe** [SainsburyWellcomeCentre/WAZP#65]
## Stupid fix:

For the moment, one can simply rerun the windows (and any other cancelled) job(s). 🔨 

## Error message:

See, for example, [this log](https://github.com/SainsburyWellcomeCentre/WAZP/actions/runs/4447357680/jobs/7808820082#step:3:828).
```
 selenium.common.exceptions.WebDriverException: Message: chrome not reachable
```

## Investigation:

Googling around it could be caused by:
* a `chromedriver` vs `chrome` incompatibility on one or more runners (!?)
* not waiting long enough,
* `start-maximised` or .. [some other option (that you would think are unrelated)](https://stackoverflow.com/questions/49205782/selenium-common-exceptions-webdriverexception-message-chrome-not-reachable-err).

## Debugging thoughts dump:

 - [x] Timeout option in `conftest.py`.
 - [x] Spit out the chrome and chromedriver versions into the log files (if they're not already there).
 - [ ] Try `start-maximised` in the `tests/test_integration/conftest.py`.
* Don't run the server integration tests on Windows jobs?

**Comment by niksirbi** [comment]
I ran into a similar error while running `pytest` locally on my (silicon) Mac:

```
ERROR tests/test_integration/test_smoke.py::test_start_server_no_errors - selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home
```

**Comment by samcunliffe** [comment]
@niksirbi that is just a missing chromedriver (unrelated to Mac silicon) ... but It's not in the dev docs so my bad.

**Comment by samcunliffe** [comment]
Just for keeping the paper trail together: @niksirbi saw [this](https://github.com/SainsburyWellcomeCentre/WAZP/actions/runs/5267489847/jobs/9522834408) which is a `WebDriverException` followed by some **_working_** tests with the chromedriver.

So it's really a timeout or startup problem (seems _not_ likely to be related to versions of things).

**Comment by samcunliffe** [comment]
Looks like this was solved by #96. Closing this for housekeeping reasons.
